back/db/migrate/20230623085057_add_votes_to_baskets_ideas.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddVotesToBasketsIdeas < ActiveRecord::Migration[7.0]
+  def change
+    add_column :baskets_ideas, :votes, :integer, null: false, default: 1
+    execute 'UPDATE ideas SET budget = 1 WHERE budget = 0'
```

Comment:
Why do we need to do this? Surely there are loads of ideas with a budget of 0 - ie it's not been set yet. Feels better to have a budget of zero than a budget of 1 as it's an indication that it's not been set correctly

Reply:
If it's so the next line doesn't fail, then maybe we should allow nil or 0 in baskets_ideas.votes?

Reply:
Actually, just checked metabase and there is just one idea with a budget of 0 and it has a basket_count of 0 too. So it's really a not an issue.

Reply:
There are however 5 ideas with a budget of -1 - could that be an issue?

Reply:
I also checked metabase and only saw one idea with budget zero. I think it was in a demo platform (not sure anymore) so the impact should be limited. I didn't check for negative values, I propose to update the update to update all ideas with a budget smaller than 1.

The reason I want to do this is because I added a validation that the votes should be larger than 0. If we don't do anything about the existing budgets, we would get inconsistent data.